sdks/python/apache_beam/pipeline_test.py

```diff
@@ -280,9 +280,10 @@ def raise_exception(exn):
         # pylint: disable=expression-not-assigned
         p | Create([ValueError]) | Map(raise_exception)
 
-  def test_eager_pipeline(self):
-    p = Pipeline('EagerRunner')
-    self.assertEqual([1, 4, 9], p | Create([1, 2, 3]) | Map(lambda x: x*x))
+  # TODO(BEAM-1894).
```

Comment:
Yes this is reproducible using a `ReadFromText` (TextSource) as well. Maybe someone who knows the python direct runner well could take a look? If it is small fix then we could wait, but if it is going to take more time then I would like to proceed without it if possible.